Examples/BoseHubbard1d/bosehubbard1d.py

```diff
@@ -18,13 +18,13 @@
 g = nk.graph.Hypercube(length=8, n_dim=1, pbc=True)
 
 # Boson Hilbert Space
-hi = nk.hilbert.Boson(graph=g, n_max=3, n_bosons=8)
+hi = nk.hilbert.Boson(N=g.n_nodes, n_max=3, n_bosons=8)
 
 # Bose Hubbard Hamiltonian
-ha = nk.operator.BoseHubbard(U=4.0, hilbert=hi)
+ha = nk.operator.BoseHubbard(hilbert=hi, graph=g.n_nodes, U=4.0)
 
 # RBM Machine with one-hot encoding, real parameters, and symmetries
-ma = nk.machine.RbmMultiVal(hilbert=hi, alpha=1, dtype=float, symmetry=True)
+ma = nk.machine.RbmMultiVal(hilbert=hi, alpha=1, dtype=float, automorphisms=g)
```

Comment:
here you should pass automorphisms=g.automorphisms ?

Reply:
Can be both